### PR TITLE
Admin CRUD product

### PR DESCRIPTION
**Pull request by bachtung144**
## Related Tickets:
- [#40596](https://edu-redmine.sun-asterisk.vn/issues/40596)

## WHAT this PR does?
admin CRUD product

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Review by hoangdv99 — APPROVED** [review]

**Comment by bachtung144** [comment]
ready

**Comment by bachtung144** [comment]
ready

File: app/Http/Livewire/Admin/AdminProductComponent.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    public function deleteProduct($id)
+    {
+        $product = Product::find($id);
+        $product->delete();
+        session()->flash('message', __('admin-product.success_delete'));
+    }
+
+    public function render()
+    {
+        $products = Product::paginate(10);
```
Comment (huyenttm-0901) [inline app/Http/Livewire/Admin/AdminProductComponent.php:22]: đưa số vào config

File: resources/views/livewire/search-component.blade.php [hunk 2]
```diff
@@ -71,6 +72,9 @@
                     </ul>
 
                 </div>
+                @else
+                    <p>No products</p>
```
Comment (huyenttm-0901) [inline resources/views/livewire/search-component.blade.php:76]: i18n

File: routes/web.php [hunk 3]
```diff
@@ -42,4 +45,8 @@
 
 Route::middleware(['auth:sanctum', 'verified'])->group(function () {
     Route::get('/admin/dashboard', AdminDashboardComponent::class)->name('admin.dashboard');
+
+    Route::get('/admin/products', AdminProductComponent::class)->name('admin.products');
+    Route::get('/admin/products/add', AdminAddProductComponent::class)->name('admin.addproduct');
+    Route::get('/admin/product/edit/{product_slug}', AdminEditProductComponent::class)->name('admin.editproduct');
```
Comment (huyenttm-0901) [inline routes/web.php:51]: group mấy route phía admin vào em

**Comment by bachtung144** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**